nucliadb_node/src/telemetry.rs

```diff
@@ -65,13 +66,46 @@ pub fn init_telemetry(settings: &Arc<Settings>) -> NodeResult<Option<ClientInitG
 }
 
 fn stdout_layer(settings: &Arc<Settings>) -> Box<dyn Layer<Registry> + Send + Sync> {
-    let format = tracing_subscriber::fmt::format().with_level(true).compact();
-
     let log_levels = settings.log_levels().to_vec();
-    tracing_subscriber::fmt::layer()
-        .event_format(format)
-        .with_filter(Targets::new().with_targets(log_levels))
-        .boxed()
+    let layer = tracing_subscriber::fmt::layer()
+        .with_level(true)
+        .with_target(true);
+
+    let filter = FilterFn::new(move |metadata| {
+        let metadata_target = String::from(metadata.target());
+
+        for (target, log_level) in log_levels.iter() {
+            let mut target = String::from(target);
+
+            if metadata.level() <= log_level {
+                // exact match or match all `*`
+                if target == metadata_target || target == ALL_TARGETS {
+                    return true;
+                }
+                // starts with
+                if target.len() > 1 && target.ends_with('*') {
+                    target.truncate(target.len() - 1);
+                    if metadata_target.starts_with(&target) {
+                        println!("true");
+                        return true;
+                    }
+                }
+            }
```

Comment:
What about moving some of this logic to `Settings`, that way we can do some previous computations that will render this filter lighter